source/stylesheets/site/_container.scss

```diff
@@ -1,5 +1,17 @@
 .container {
   @include margin(null auto);
-  @include padding(null $base-spacing);
-  max-width: 42em;
+  flex: 1 50%;
+  padding: $base-spacing;
+
+  @media (min-width: $medium-screen) {
+    padding: $large-spacing;
+  }
+
+  @media (min-width: 1000px) {
+    padding: $x-large-spacing;
+  }
+
+  .banner & {
```

Comment:
Instead of having `.banner` styles in multiple locations (`_banner.scss`), thoughts on doing something like `container--narrow`?